dev/com.ibm.ws.jsonb.cdi/src/com/ibm/ws/jsonb/cdi/JsonbBean.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jsonb.cdi;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.spi.CreationalContext;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Default;
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.InjectionPoint;
+import javax.enterprise.inject.spi.PassivationCapable;
+import javax.json.bind.Jsonb;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.ras.annotation.Trivial;
+
+@Trivial
+public class JsonbBean implements Bean<Jsonb>, PassivationCapable {
+
+    private static final TraceComponent tc = Tr.register(JsonbBean.class);
+
+    static final Set<Type> types = new HashSet<Type>(Arrays.asList(Object.class, Jsonb.class));
+    static final Set<Annotation> qualifiers = new HashSet<Annotation>(Arrays.asList(Default.Literal.INSTANCE, Any.Literal.INSTANCE));
+
+    @Override
+    public Jsonb create(CreationalContext<Jsonb> ctx) {
+        return JsonbExtension.create(ctx);
```

Comment:
Rather than static reference to OSGi service component, get it from service registry.